client/systems.go

```diff
@@ -221,6 +235,17 @@ type InstallSystemOptions struct {
 	// OnVolumes is the volume description of the volumes that the
 	// given step should operate on.
 	OnVolumes map[string]*gadget.Volume `json:"on-volumes,omitempty"`
+	// OptionalInstall contains the optional snaps and components that should be
+	// installed on the system. Omitting this field will result in all optional
+	// snaps and components being installed.
```

Comment:
According to SD168 it is the other way around: "If the “snaps” and “components” fields are empty and “all” is false or omitted, no optional snaps or components are installed"

Reply:
There is a distinction between omitting the entire `optional-install` field (which is what the comment is about) and leaving everything inside of `optional-install` empty (which is what that part of the spec is talking about).

Reply:
Ok, I see - maybe it would be worth it to have this information as part of this comment too to make it easy to remember.